src/cogent3/core/annotation_db.py

```diff
@@ -72,14 +76,11 @@ class FeatureDataType(typing.TypedDict):
 
 @typing.runtime_checkable
 class SerialisableType(typing.Protocol):  # pragma: no cover
-    def to_rich_dict(self) -> dict:
-        ...
+    def to_rich_dict(self) -> dict: ...
 
-    def to_json(self) -> str:
-        ...
+    def to_json(self) -> str: ...
```

Comment:
eek! this looks like a black version issue. Can you please check you run the current pinned version against the repo, it should revert these.

Reply:
Whoops! Looks like vscode is trying to use the right version, but something else is going on. I'll run the commands manually for now.